src/compiler/compiler.rs

```diff
@@ -542,6 +549,9 @@ gcc
     let mut cmd = creator.clone().new_command_sync(&executable);
     cmd.stdout(Stdio::piped())
        .stderr(Stdio::null());
+    for &(ref k, ref v) in env.iter() {
```

Comment:
We do have [`RunCommand::envs`](https://github.com/mozilla/sccache/blob/master/src/mock_command.rs#L110) FYI.

Reply:
Sure thing, updated!